### PR TITLE
[DCOS-57593][HDFS] Add base tech cli commands

### PR DESCRIPTION
**Pull request by alexeygorobets**
Added new command `hdfs` that allows to run HDFS CLI commands via DC/OS CLI
```
$ dcos hdfs hdfs dfs -mkdir /dir
$ dcos hdfs hdfs dfs -ls /
Found 1 items
drwxr-xr-x   - nobody supergroup          0 2019-09-02 10:11 /dir
```
List of HDFS CLI commands: https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HDFSCommands.html

- added new command to hdfs framework - frameworks/hdfs/cli/main.go, remaining arguments after `hdfs` command will be considered as subcommands and flags for HDFS CLI
- added building CLI tool as part HDFS framework build
- `test_cli.py` test creates a directory in HDFS via command line commands and verifys it

### CODE REVIEW

File: frameworks/hdfs/cli/main.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+type HdfsCliArgs struct {
+	arguments []string
+}
+
+func (args *HdfsCliArgs) String() string {
+	return strings.Join(args.arguments, " ")
+}
+
+func (args *HdfsCliArgs) runHdfs(a *kingpin.Application, e *kingpin.ParseElement, c *kingpin.ParseContext) error {
+	dcosExecCommand := []string{"task", "exec", "name-0-node", "bash", "-c"}
```
Comment (samvantran) [inline frameworks/hdfs/cli/main.go]: I think this is fine for now but eventually we'd probably want to make it possible to target specific hdfs nodes. For example, depending on which namenode you run the command `hdfs haadmin -getServiceState` on, you'd get `active|standby`. 

Thinking it over, is it easy to add a flag for targeting a certain node and default to `name-0-node` if no flag provided?

File: frameworks/hdfs/tests/test_cli.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.mark.sanity
+def test_cli():
+    sdk_cmd.run_cli("package install {} --yes --cli".format(config.PACKAGE_NAME))
+    dirpath = "/test"
+    sdk_cmd.svc_cli(config.PACKAGE_NAME, config.SERVICE_NAME, "hdfs dfs -mkdir {}".format(dirpath))
+    _, stdout, _ = sdk_cmd.svc_cli(config.PACKAGE_NAME, config.SERVICE_NAME, "hdfs dfs -ls /")
```
Comment (samvantran) [inline frameworks/hdfs/tests/test_cli.py]: I'd say let's add more tests to verify other commands work as expected but given team's current priorities, I think this is fine for now

Comment (alexeygorobets) [reply frameworks/hdfs/tests/test_cli.py]: I will think about corner cases that can be tested, thanks!

File: frameworks/hdfs/cli/main.go (hunk 1 above)
Comment (alexeygorobets) [reply frameworks/hdfs/cli/main.go]: yes, thanks for the suggestion! will do that

File: frameworks/hdfs/cli/main.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	hdfsCommand := fmt.Sprintf("export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/); $HDFS_VERSION/bin/hdfs %s", args.String())
+	fullCommand := append(dcosExecCommand, hdfsCommand)
+
+	value, _ := client.RunCLICommand(fullCommand...)
```
Comment (zmalik) [inline frameworks/hdfs/cli/main.go]: you can use the `error` here to return it if it's not `nil`
The method is returning an error but that value isn't used anywhere. So just print the error also here in case it's not nil

File: frameworks/hdfs/cli/main.go [hunk 4]
```diff
@@ -0,0 +1,74 @@
+package main
```
[the remark this answers is not in the document]
Comment (alexeygorobets) [reply frameworks/hdfs/cli/main.go]: thank you! forgot about this awesome tool

File: frameworks/hdfs/cli/main.go (hunk 3 above)
Comment (alexeygorobets) [reply frameworks/hdfs/cli/main.go]: done!

**Comment by samvantran** [comment]
Just found out one thing we missed was adding a message in the `dcos hdfs -h` about this new feature!
